molli/chem/structure.py

```diff
@@ -142,12 +142,18 @@ def yield_from_mol2(
                 res.coords[i] = a.xyz
                 res.atoms[i].set_mol2_type(a.mol2_type)
                 res.atoms[i].label = a.label
+                # This is to take care of tripos atom charge block
+                if chrg := a.attrib.pop("charge", None):
```

Comment:
I think it would be worth adding a comment here. I agree that it should be set to None so that any errors within mol2_type can be tracked rather than defaulting to 0, it should just be referenced somewhere in the code.